include/sys/syscall.h

```diff
@@ -540,27 +540,29 @@
 #  define SYS_sendto                   (__SYS_network + 10)
 #  define SYS_setsockopt               (__SYS_network + 11)
 #  define SYS_socket                   (__SYS_network + 12)
+#  define __SYS_socket                 (__SYS_network + 13)
 #else
-#  define SYS_socket                    __SYS_network
+#  define __SYS_socket                    __SYS_network
```

Comment:
Fix vertical alignment of righthand column

Reply:
I got. Thanks.